_posts/2022-09-21-ballerina-is-moving-from-slack-to-discord.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: blog-post
+title: Ballerina is Moving from Slack to Discord
+author: Ballerina Team
+published-date: 21 September 2022
+status: Published
+socialmediaimage: Ballerina-Swan-Lake-GA-Release-banner-02-with-button.png
+permalink: /posts/2022-09-21-ballerina-is-moving-from-slack-to-discord/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+We will be moving from Slack to [Discord](https://discord.com/invite/wAJYFbMrG2){:target="_blank"} from Saturday, October 1. The WSO2 Slack community has grown to over 7,000 users, and we are excited to have them meet our 1000+ users on Discord! Join the Ballerina server by clicking [here](https://discord.com/invite/wAJYFbMrG2){:target="_blank"}.
```

Comment:
IMO, instead of `WSO2 Slack community` we need to talk about Ballerina slack community here.

Reply:
Resolved